pyproject.toml

```diff
@@ -42,7 +42,6 @@ exclude = [
     'core/config_peer\.py', # 1
     'core/evaluation_parameters\.py', # 6
     'core/expectation_configuration\.py', # 21
-    'core/expectation_suite\.py', # 10
```

Comment:
👏

Reply:
1 last typing issue. LSP violations strike again 😄 
```
great_expectations/data_context/data_context/file_data_context.py:84: error: Argument 4 of "save_expectation_suite" is incompatible with supertype "AbstractDataContext"; supertype defines the argument type as "Optional[bool]"  [override]
great_expectations/data_context/data_context/file_data_context.py:84: note: This violates the Liskov substitution principle
great_expectations/data_context/data_context/file_data_context.py:84: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
``

Reply:
Hi @Kilo59 this is still WIP, I will update you when it is ready for review.